morebits.js

```diff
@@ -1401,14 +1507,14 @@ Morebits.unbinder.getCallback = function UnbinderGetCallback(self) {
 
 
 
+/* **************** Morebits.date **************** */
```

Comment:
I kept these non-standard headers for internal searching/jumping, but I don't actually think they're necessary to keep around.